docs/chain/collator.md

```diff
@@ -530,8 +530,18 @@ For **peregrine**, the parachain bootnodes are:
 
 ## Lifecycle of a Collator
 
-<Mermaid
-chart={`flowchart TD
+When you intent to stop collating or just being a collator candidate, you have to go through three stages until your staked tokens are unlocked and your collator state is purged from the chain.
+
+First, you signal your intent by calling `init_leave_candidates`. You will both be removed from the `CandidatePool` and your state switches from `Active` to `Leaving(leave_round)`, where `leave_round` reflects the number of sessions in which you can actually leave. A leaving candidate cannot be selected as an active collator for the sessions hereinafter. Moreover, you cannot receive new Delegations and existing Delegations cannot be adjusted. However, Delegations can still be revoked.
+
+Second, after waiting until the `leave_round`-th session, you can call `execute_leave_candidate` to remove all of your `Candidate` associated storage. You should be certain as there is no turning back afterwards. If you wish to become a Candidate at a later stage, you have to apply again and miss out on all of your former delegations.
+
+However, you can still cancel your exit request by calling `cancel_leave_candidates` if you have not exited yet and the CandidatePool is not full already. Upon cancelling your exit intent, your state switches back to `Active` and you still have all Delegations which were not revoked in the meantime. Moreover, if you are you are one of the top staked candidates, you will automatically become a Collator in two rounds (~4 hours).
+
+In case you executed the exit request, you cannot immediately unlock your previously staked tokens. There is a delay of 7 days in block time before you can free them by calling `unlock_unstaked`. 
```

Comment:
Very minor nit pick. I try to enforce that we have a one sentence per line policy. That makes it easier for git to highlight changes and for reviewers to address individual sentences.

```suggestion
In case you executed the exit request, you cannot immediately unlock your previously staked tokens.
There is a delay of 7 days in block time before you can free them by calling `unlock_unstaked`.
```